website/src/views/NotFound/NotFound.test.jsx

```diff
@@ -11,4 +18,15 @@ describe('NotFound View', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot(); // eslint-disable-line no-undef
```

Comment:
You can remove these comments for disabling eslint they have been resolved in the eslint config